docs/source/conf.py

```diff
@@ -60,12 +60,18 @@ def setup(sphinx):
     }, True)
     sphinx.add_transform(AutoStructify)
 
-
+current_version = "3.1.0"
+res = ""
+if os.path.isfile('../../CURRENT_VERSION.sh'):
+    with open('../../CURRENT_VERSION.sh', 'r') as file:
+        current_version = file.read().replace('\n', '')
+current_branch = 'branch-' + '.'.join(current_version.split('.')[:2])
```

Comment:
``CURRENT_VERSION.sh`` is not tracked in the repository.

Reply:
it does in the branches it's not on master, that's why the: `if os.path.isfile('../../CURRENT_VERSION.sh')` is for